.github/workflows/nessie.yaml

```diff
@@ -103,6 +103,63 @@ jobs:
             docker push $ECR_REGISTRY/$ECR_REPOSITORY_LAKEFS:${{ steps.version.outputs.tag }}
           fi
 
+  spark:
+    name: Test lakeFS with Spark
+    needs: deploy-image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check-out code
+        uses: actions/checkout@v2
+
+      - name: Extract version
```

Comment:
Can this be a workflow level var? This is the third time it runs in this workflow.